### PR TITLE
Adds a test based on a failure to notify api

### PR DESCRIPTION
**Pull request by sam-drumm**
https://hackney.atlassian.net/browse/TS-1783?atlOrigin=eyJpIjoiNzM1NmIyZWY1OWNmNGFmNThlZDRlNDZiODgwMDM1MWYiLCJwIjoiaiJ9

**What**
- Updates test to ensure if the notify email fails, it will still successfully call and complete the disqualification action. 
- Updates local e2e test to use common function.

### CODE REVIEW

**Review by LBHTKarki — APPROVED** [review]

File: cypress/e2e/pages/apply/[resident]/summary.cy.ts [hunk 1]
```diff
@@ -50,11 +51,39 @@ describe('Apply resident summary page', () => {
...
+      false
+    );
+
+    cy.mockHousingRegisterApiGetApplications(
+      applicationId,
+      ineligibleApplicationWithCompletedMainApplicantSections,
+      true
+    );
+
+    cy.mockNotifyEmailResponse(StatusCodes.INTERNAL_SERVER_ERROR);
```
Comment (Duslerke) [inline cypress/e2e/pages/apply/[resident]/summary.cy.ts:72]: Is there any user feedback that this failure happened?

File: cypress/e2e/pages/apply/[resident]/summary.cy.ts [hunk 2]
```diff
@@ -50,11 +51,39 @@ describe('Apply resident summary page', () => {
...
+    ApplyExpectPage.getContinueToNextStepButton().click();
+    ApplyOverviewPage.getApplicantButton(personId).click();
+    ApplyResidentSummaryPage.getConfirmDetailsButton().click();
+
+    RejectionPage.getRejectionPage().should('be.visible');
   });
 
   it('shows saving message when user confirms the resident details and is not eligible to apply', () => {
     //Application object does not reflect the correct state after patch, but it doesn't matter for this test
+    cy.mockNotifyEmailResponse();
```
Comment (Duslerke) [inline cypress/e2e/pages/apply/[resident]/summary.cy.ts:85]: What does empty mock represent?

File: cypress/e2e/pages/apply/[resident]/summary.cy.ts (hunk 1 above)
Comment (sam-drumm) [reply cypress/e2e/pages/apply/[resident]/summary.cy.ts:72]: These are all (notify) currently silent failures. At this point, we're going to leave it that way to be addressed later as a known issue. 

This test was really just to ensure that in the try block a failure of that promise wouldn't stop the next action. As it's not unwrapped it's assumed as much but just wanted to make sure in a test.

File: cypress/e2e/pages/apply/[resident]/summary.cy.ts (hunk 2 above)
Comment (sam-drumm) [reply cypress/e2e/pages/apply/[resident]/summary.cy.ts:85]: Default is a 200 response.

**Pull request merged**